CHANGELOG.md

```diff
@@ -10,8 +10,9 @@
     - Local backups support saving the GraphDB backup archives in volume from an existing persistent volume claim, configured
       with `backup.local`
     - Cloud backups support uploading the GraphDB backup archives in one of the supported cloud object storage services, configured
-      with `backup.cloud`
     - Added a new example under [examples/backup-local](examples/backup-local) showing how to use the local backup feature
+      with `backup.cloud`
+- Added a new property - `security.provisioner.passwordHash`
```

Comment:
It's not very helpful like this, at least add a short description of what it does/how it helps.